test/cmd/events.sh

```diff
@@ -42,6 +42,62 @@ run_kubectl_events_tests() {
     kube::test::get_object_assert 'cronjob --namespace=test-events' "{{range.items}}{{ if eq $id_field \"pi\" }}found{{end}}{{end}}:" ':'
     ### Create a cronjob in a specific namespace
     kubectl create cronjob pi --schedule="59 23 31 2 *" --namespace=test-events "--image=$IMAGE_PERL" -- perl -Mbignum=bpi -wle 'print bpi(20)' "${kube_flags[@]:?}"
+    ### Create a crd
+    kubectl create -f - << __EOF__
+{
+  "kind": "CustomResourceDefinition",
+  "apiVersion": "apiextensions.k8s.io/v1",
+  "metadata": {
+    "name": "cronjobs.example.com"
+  },
+  "spec": {
+    "group": "example.com",
+    "scope": "Namespaced",
+    "names": {
+      "plural": "cronjobs",
+      "singular": "cronjob",
+      "kind": "Cronjob",
+      "shortNames": ["mycj"]
```

Comment:
I think, no need to define a shortName and instead we can delete it with it's full name.

Reply:
Yes, that makes sense. I added it for convenience when testing, but it's not necessary.